cgroups/src/common.rs

```diff
@@ -60,6 +61,23 @@ impl Display for CgroupSetup {
     }
 }
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum FreezerState {
+    Undefined,
+    Frozen,
+    Thawed,
+}
+
+#[derive(Clone, Debug, Default)]
+pub struct ControllerOpt {
```

Comment:
Can I ask you to comment on the description of this structure using doc comments?
https://doc.rust-lang.org/rust-by-example/meta/doc.html

Reply:
OK. I pushed this change.